src/emmy/value.cljc

```diff
@@ -331,7 +246,7 @@
 
      IPrintWithWriter
      (-pr-writer [x writer _]
-       (let [rep (if (<= x (.-MAX_SAFE_INTEGER js/Number))
+       (let [rep (if (< (if (< x 0) (- x) x) (.-MAX_SAFE_INTEGER js/Number))
```

Comment:
good catch! I might recommend a test for this if it didn't make it in yet, which it might have, apologies for my code nav on this PR

Reply:
Something definitely breaks w/o this :) 

What has happened, I think, is that there were some "overlapping" cases in the Value regime that have been clarified by the transition to defmulti, which focused my attention on the (very few) bugs that I found